### PR TITLE
make KubeadmControlPlane name indedependent from cluster-aws version

### PR DESCRIPTION
**Pull request by bdehri**
### Checklist

- [x] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

**Comment by AverageMarcus** [comment]
(I need to test something with the E2E tests, please ignore) 

/run cluster-test-suites

**Review by AndiDog — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- make AWS instances independent of helm label to prevent unnecessary rolling.
```
Comment (fiunchinho) [inline CHANGELOG.md:12]:
```suggestion
- Make AWS instances names independent of helm label to prevent unnecessary rolling.
```

**Review by fiunchinho — APPROVED** [review]

**Comment by AverageMarcus** [comment]
Sorry, I needed to manually cancel the in-progress test run. Re-triggering...


/run cluster-test-suites

**Pull request merged**